libs/judicial-system/types/src/lib/institution.ts

```diff
@@ -16,4 +16,5 @@ export interface Institution {
   defaultCourtId?: string
   policeCaseNumberPrefix?: string
   nationalId?: string
+  notificationEmail?: string
```

Comment:
This is not needed unless you add the new field to the GraphQL api for use in the client. Consider removing this or adding the field the GraphQL api.